lib/index.js

```diff
@@ -150,7 +153,14 @@ module.exports = function(script, scriptArgs, nodeArgs, opts) {
 
     // Upon errors, display a notification and tell the child to exit.
     ipc.on(child, 'error', function(m) {
-      log.debug('Child error')
+      log.debug('Child error', m)
+      var lastRequire = m.lastRequire
+      if (m.code === 'MODULE_NOT_FOUND' && lastRequire && lastRequire.path.startsWith('.')) {
+        var pathNoExt = path.normalize(path.join(path.dirname(lastRequire.filename), lastRequire.path))
+        var watch = chokidar.watch([pathNoExt + '.ts', pathNoExt + '.tsx'])
```

Comment:
why not just call `watcher.add()` here? What does chokidar do that `filewatcher()` cant?

Reply:
@jonjaques Please enlighten me if I missed something, but `watcher.add()` didn't work with non-existant files when I tried

Reply:
Naw, was legit curious. 

As I understand it this PR is to handle the case where you're in an existing file and you import something that doesn't yet exist:

import {something} from './nonexistent'

This handles the error that would shut down ts-node-dev and then sets up a watcher for what would be `./nonexistent.ts|tsx`.

I guess what I meant was, both `filewatcher` and `chokidar` are wrappers around the FS.watch, it just seems like conceptually this project should use one or the other. 

So different question— could we switch all the `filewatcher` code to use `chokidar` instead?  Last publish on filewatcher was 3 years ago.

Reply:
I'm all for doing that @jonjaques! I think chokidar is more mature

However, I'm no official contributor. Could coordinate if @whitecolor made me